### PR TITLE
Port windows.ui.xaml.textalignment to Avalonia

### PR DESCRIPTION
**Pull request by Gillibald**
## Checklist

- [ ] Added unit tests (if possible)?
- [x] Added XML documentation to any related classes?
- [ ] Consider submitting a PR to https://github.com/AvaloniaUI/Documentation with user documentation

### CODE REVIEW

File: src/Avalonia.Base/Media/TextFormatting/TextLine.cs [hunk 1]
```diff
@@ -218,24 +218,14 @@ public abstract class TextLine
                         return Math.Max(0, (paragraphWidth - width) / 2);
 
                     case TextAlignment.Right:
-                        return Math.Max(0, paragraphWidth - widthIncludingTrailingWhitespace);
+                        return flowDirection == FlowDirection.LeftToRight ? Math.Max(0, paragraphWidth - widthIncludingTrailingWhitespace) : 0;
```
Comment (jp2masa) [inline src/Avalonia.Base/Media/TextFormatting/TextLine.cs:221]: I think this `switch` was supposed to be moved outside the `if`? (i.e. removing the `if`)

Comment (Gillibald) [reply src/Avalonia.Base/Media/TextFormatting/TextLine.cs:221]: Alignment is inverted for RightToLeft

Comment (jp2masa) [reply src/Avalonia.Base/Media/TextFormatting/TextLine.cs:221]: Right, but this switch is inside an `if (flowDirection == FlowDirection.LeftToRight)`, and now it's handled in the ternary, I think you forgot to remove it?

Comment (Gillibald) [reply src/Avalonia.Base/Media/TextFormatting/TextLine.cs:221]: I will just invert the Alignment value to make it more readable and more compact

**Comment by avaloniaui-team** [comment]
You can test this PR using the following package version. `0.10.999-cibuild0021256-beta`. (feed url: https://nuget.avaloniaui.net/repository/avalonia-all/index.json) [PRBUILDID]

**Review by grokys — APPROVED** [review]

**Comment by avaloniaui-team** [comment]
You can test this PR using the following package version. `0.10.999-cibuild0021273-beta`. (feed url: https://nuget.avaloniaui.net/repository/avalonia-all/index.json) [PRBUILDID]

**Comment by avaloniaui-team** [comment]
You can test this PR using the following package version. `0.10.999-cibuild0021288-beta`. (feed url: https://nuget.avaloniaui.net/repository/avalonia-all/index.json) [PRBUILDID]

**Comment by avaloniaui-team** [comment]
You can test this PR using the following package version. `0.10.999-cibuild0021389-beta`. (feed url: https://nuget.avaloniaui.net/repository/avalonia-all/index.json) [PRBUILDID]

**Pull request merged**